include/motor-control/core/stepper_motor/tmc2160_driver.hpp

```diff
@@ -295,9 +315,9 @@ class TMC2160 {
      * be read, so this function gets it from the actual device.
      */
     auto update_glob_scaler(uint32_t data) -> void {
-        auto ret = read_register<GConfig>(data);
+        auto ret = read_register<GConfig>(data);  // should be GlobalScalar
```

Comment:
yeah it really should be huh. @ahiuchingau what do you think of this? Might impact some current-control stuff. @pmoegenburg would you mind opening a second quick PR to fix this and removing the comments here because otherwise they'll cause merge conflicts? Great find.